tests/benchdnn/graph/ref_partition.cpp

```diff
@@ -322,6 +322,7 @@ int ref_partition_t::check_partition_correctness(
         size_t op_id = op.get().id_;
         const auto op_kind = op.get().kind_;
         const auto ref_prim = ref_prims_.at(op_id);
+        std::string mode {};
```

Comment:
std::string has default ctor.
```suggestion
        std::string mode;
```